test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -39,6 +39,7 @@ Feature: Bind a single application to multiple services
             metadata:
                 name: binding1-myapp-2-sbrs
             spec:
+                bindAsFiles: false
```

Comment:
do we have now an acceptance where `bindAsFiles` is unset? to check the default behavior?

Reply:
I have mentioned the exact test in which `bindAsFiles` is unset on the PR description